data/mods/Aftershock/player/professions.json

```diff
@@ -186,5 +186,38 @@
       "male": [ "briefs", "collarpin", "diamond_ring" ],
       "female": [ "panties", "hairpin", "pearl_collar" ]
     }
+  },
+  {
+    "type": "profession",
+    "ident": "afs_mastadon_swat",
+    "name": "Upifted SWAT Officer",
+    "description": "Uplifted elephants are created with human, elephant and mastodon DNA among others.  Exclusively created by the military they are often known as Killophants.  You served your tours and earned the right to be considered a full citizen.  While many of your peers became mercenaries, you decided to devote your life to helping others.  Now you may be the last lawman standing, and stand you will against the hordes.",
+    "points": 10,
+    "skills": [
+      { "level": 3, "name": "gun" },
+      { "level": 3, "name": "shotgun" },
+      { "level": 1, "name": "pistol" },
+      { "level": 1, "name": "melee" },
+      { "level": 1, "name": "bashing" }
+    ],
+    "traits": [ "PROF_SWAT", "THRESH_MASTODON" ],
```

Comment:
You are still missing the other mutations here, I think?

Reply:
Waiting on Mutant NPCs to merge so I can try and make the profession load with random mutations.

Reply:
Ah got it.